webpack/webpack.dev.js

```diff
@@ -4,10 +4,27 @@ const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const InterpolateHtmlPlugin = require('interpolate-html-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const MonacoWebpackPlugin = require('monaco-editor-webpack-plugin');
 const common = require("./common");
-// const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
-// const OptimizeCssAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+
+// extended configuration template
+let extendedConfig = {
+	entry: {},
+	output: {},
+	extraPlugins: [],
+	optimization: {},
+	module: {
+		extraRules: []
+	}
+};
+
+// load extended configuration
+try {
+	extendedConfig = { ...extendedConfig, ...require("../../asab-webui.config") } ;
```

Comment:
Can you please add comment why you have a relative path here?

Reply:
@Pe5h4 Hi Franta,
I actually do not understand what do you mean by why do I have it there